src/test/common/ios/plistBuddy.test.ts

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for details.
+
+import {PlistBuddy} from "../../../common/ios/plistBuddy";
+
+import * as assert from "assert";
+import * as Q from "q";
+import * as sinon from "sinon";
+
+suite("plistBuddy", function() {
+    suite("commonContext", function() {
+        test("setPlistProperty should attempt to modify, then add, plist properties", function() {
+            const plistFileName = "testFile.plist";
+            const plistProperty = "myProperty";

```

Comment:
I know. But if I'm going to test a single value, knowing that :RCTDevMenu:executorClass works gives me a lot more confidence than the fact that myProperty works. We should try to test the most important case for us, rather than a random case. The code could potentially break if you put a : on the string, or something else that happens with :RCTDevMenu:executorClass but not with myProperty.
On the other hand if myProperty breaks because it doesn't has a :, and we don't detect that, it won't cause any paint for the users, so it doesn't really matter that much.